tasks/get_sql_logins.ps1

```diff
@@ -124,9 +39,7 @@ try {
   $SQLInstances = Get-SQLInstances -instance_name $instance_name
 }
 catch {
-  $error._error.msg = 'Cannot detect SQL instance names.'
-  $error._error.details.detailedInfo = $_
-  return $error | ConvertTo-JSON
+  return Write-BoltError 'Cannot find SQL instances' $_
```

Comment:
Should the value after return be in parentheses?

Reply:
Doesn't need to be, unless you are suggesting stylistically that it should be.